crates/holochain/src/sweettest/sweet_conductor_config.rs

```diff
@@ -80,7 +80,11 @@ impl SweetConductorConfig {
             })
             .tune_conductor(|tune| {
                 tune.sys_validation_retry_delay = Some(std::time::Duration::from_secs(1));
-            })
+            });
+
+        // Allow device seed generation to exercise key derivation in sweettests.
+        c.device_seed_lair_tag = Some("sweet-conductor-device-seed".to_string());
+        c.danger_generate_throwaway_device_seed = true;
```

Comment:
Making this the default is fine with me but the only other test change I see is testing reading conductor configs. Does that mean we don't have a test which uses DPKI the way it would be without this test feature turned on? Or are there some tests like that which just didn't need to change?

Reply:
Correct, we don't, but the only difference between this and the "real" way is that the device seed would already be created in lair prior to conductor startup. Everything else is the same.

Reply:
Without a test, I could modify the code and accidentally always overwrite the seed in Lair - that would be really bad and unexpected behaviour. So even if it's currently right, I'd like to be verifying the behaviour in both cases

Reply:
I could add a test, but I actually don't know what the test would be. Just a check that an app can be installed with this flag being unset? I get that we want to test every codepath, but I don't know what we would test *about* the other path, other than that it succeeds.

Note that Lair [doesn't](https://github.com/holochain/lair/blob/265256449a095e13e519d5f5e44f09bb43e9a108/crates/lair_keystore_api/src/mem_store.rs#L116-L119) allow [overwriting](https://github.com/holochain/lair/blob/9df5161f28e2f5a6b26b8887441d923ad67e4823/crates/lair_keystore/src/sql/schema.sql#L8) keys.